xml/issue3753.xml

```diff
@@ -253,7 +253,8 @@ It also partially addresses GB-130 (along with LWG <iref ref="3814"/>).
 
 </discussion>
 
-<resolution>
+<strong>Previous resolution [SUPERSEDED]:</strong>
```

Comment:
N.B. the previous resolution should be within the `<discussion>` element, not after it.

Reply:
I see, thanks. Could we get tag for previous resolution :)

Reply:
Done: https://github.com/cplusplus/LWG/commit/26ab54e0d68fc719081b5b9e7946ea3fa9e4e1f9

Use `<superseded>` to enclose the previous resolution (but it still needs to be inside the `<discussion>` not outside it!)

The start element will automatically insert:

```
<p><strong>Previous resolution [SUPERSEDED]:</strong></p>
<blockquote class="note">
```

and the close element will add `</blockquote>`.